File: sdkwork-ai-sdk-csharp/Models/EmbeddingResponse.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class EmbeddingResponse
    {
        public string? Object { get; set; }
        public List<Embedding>? Data { get; set; }
        public string? Model { get; set; }
        public Usage? Usage { get; set; }
    }
}

File: sdkwork-ai-sdk-typescript/src/types/embedding-response.ts
import type { Embedding } from './embedding';
import type { Usage } from './usage';

/** OpenAI标准嵌入响应 */
export interface EmbeddingResponse {
  /** The object type, which is always 'list' */
  object?: string;
  /** The list of embeddings */
  data?: Embedding[];
  /** The model used for generating embeddings */
  model?: string;
  /** Usage statistics for the request */
  usage?: Usage;
}
